src/core/client/stream/tabs/Comments/ReplyList/ReplyList.spec.tsx

```diff
@@ -1,42 +1,70 @@
 import { shallow } from "enzyme";
-import { noop } from "lodash";
 import React from "react";
+import { Environment, RecordSource } from "relay-runtime";
 import sinon, { SinonSpy } from "sinon";
 
-import { removeFragmentRefs } from "coral-framework/testHelpers";
+import {
+  createRelayEnvironment,
+  removeFragmentRefs,
+} from "coral-framework/testHelpers";
 import { PropTypesOf } from "coral-framework/types";
+import { initLocalState } from "coral-stream/local";
 
 import ReplyList from "./ReplyList";
 
-const ReplyListN = removeFragmentRefs(ReplyList);
+let environment: Environment;
+let source: RecordSource;
 
-it("renders correctly", () => {
-  const props: PropTypesOf<typeof ReplyListN> = {
-    story: { id: "story-id" },
-    comment: { id: "comment-id" },
-    comments: [
-      { id: "comment-1" },
-      { id: "comment-2", showConversationLink: true },
-    ],
-    onShowAll: noop,
-    hasMore: false,
-    disableShowAll: false,
-    indentLevel: 1,
-    viewer: null,
-    localReply: false,
-    disableReplies: false,
-    settings: {
-      reaction: {
-        icon: "thumb_up_alt",
-        label: "Respect",
-      },
-    },
-  };
-  const wrapper = shallow(<ReplyListN {...props} />);
-  expect(wrapper).toMatchSnapshot();
+const context = {
+  localStorage: window.localStorage,
+  sessionStorage: window.sessionStorage,
+  window,
+  renderWindow: window,
+};
+
+beforeEach(() => {
+  source = new RecordSource();
+  environment = createRelayEnvironment({
+    source,
+    initLocalState: false,
+  });
 });
 
-describe("when there is more", () => {
+const ReplyListN = removeFragmentRefs(ReplyList);
+
+// TODO (Nick): This will need to be rewritten with a full testRenderer as an
+//   integration test for this to work. We can't do shallow and have ReplyList
+//   use local lookups.
+//
+// it("renders correctly", async () => {
+//   await initLocalState({ environment, context: context as any });
+//   const props: PropTypesOf<typeof ReplyListN> = {
+//     story: { id: "story-id" },
+//     comment: { id: "comment-id" },
+//     comments: [
+//       { id: "comment-1" },
+//       { id: "comment-2", showConversationLink: true },
+//     ],
+//     onShowAll: noop,
+//     hasMore: false,
+//     disableShowAll: false,
+//     indentLevel: 1,
+//     viewer: null,
+//     localReply: false,
+//     disableReplies: false,
+//     settings: {
+//       reaction: {
+//         icon: "thumb_up_alt",
+//         label: "Respect",
+//       },
+//     },
+//   };
+//   const wrapper = shallow(<ReplyListN {...props} />);
+//   expect(wrapper).toMatchSnapshot();
+// });
```

Comment:
I'm not rewriting this whole test into an integration test right now. We want to get rid of all these `shallow` render test anyways as they're very brittle and break any time any local state or relay changes happen due to the fact that all fragments are removed from the props when we create `ReplyListN` for this test.

Reply:
Sounds good, and thanks for leaving the comment with it.

Reply:
Actually, I found a way to fix it, test is updated and passing!